source/fundamentals/builders/aggregation-expressions.txt

```diff
@@ -1,6 +1,6 @@
-=================================
-Aggregation Expression Operations
-=================================
+=======================
+Aggregation Expressions
```

Comment:
Question:

Is this the correct term? From our initial discussions with Maxim, I thought we wanted to call them "Aggregation Expression Operators"

Reply:
You are correct - since the ticket changed the name, I thought that was what it was supposed to be, but that was not correct.